mewarpx/changelog.csv

```diff
@@ -1,5 +1,5 @@
 Version, Physics version, Date,        List of changes
-8.0.0, 2, In progress, "
+8.1.0, 2, In progress, "
```

Comment:
We should leave the 8.0.0 record below so instead of changing that entry in the changelog, please add a new entry for version 8.1.0. It is probably worth it for us to merge the bug fix in #171 before making this fix though.